pmda/test/test_rdf.py

```diff
@@ -83,6 +83,15 @@ def test_same_result(sels, n_blocks):
     assert_almost_equal(nrdf.rdf, prdf.rdf)
 
 
+def test_cdf(sels):
+    s1, s2 = sels
+    rdf = InterRDF(s1, s2).run()
+    rdf.get_cdf()
```

Comment:
That's weird use of a `get_*` "function" – if anything, I'd expected
```python
cdf = rdf.get_cdf()
```
It's very confusing if then later magically an attribute `rdf.cdf` appears.